### PR TITLE
Expose backlight mode for TuYa - TS0004 - _TZ3000_nivavasg

### PR DESCRIPTION
**Pull request by andre991**
Expose backlight mode to AVATTO - TuYa - TS0004 - 

_TZ3000_nivavasg
_TZ3000_gexniqbq

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -3740,6 +3740,34 @@ const definitions: Definition[] = [
...
         },
     },
+    {
+        fingerprint: [
+            {modelID: 'TS0004', manufacturerName: '_TZ3000_nivavasg'},
+        ],
+        model: 'TS0004_switch_4_gang_with_backlight',
+        vendor: 'TuYa',
+        description: 'Smart light switch - 4 gang with backlight and neutral wire',
+        extend: tuya.extend.switch({powerOnBehavior2: true, backlightModeOffOn: true, endpoints: ['l1', 'l2', 'l3', 'l4']}),
```
Comment (Koenkk) [inline src/devices/tuya.ts:3750]: Can you add this to the existing `TS0004` instead?

Comment (andre991) [reply src/devices/tuya.ts:3751]: Yes I can, I just made that way because I see others PR's creating a new one.

I'll add TS0004.

Comment (andre991) [reply src/devices/tuya.ts]: @Koenkk I updated the PR using the existent TS0004.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**